test/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.test.js

```diff
@@ -154,7 +226,12 @@ describe('Fetch Bill Run Licences service', () => {
 
   describe('when there is an invalid bill run id passed to the service', () => {
     it('returns no results', async () => {
-      const result = await FetchBillRunLicencesService.go('56db85ed-767f-4c83-8174-5ad9c80fd00d', undefined, undefined)
+      const result = await FetchBillRunLicencesService.go(
+        '56db85ed-767f-4c83-8174-5ad9c80fd00d',
+        undefined,
```

Comment:
Here you have put undefined in the 'FetchBillRunLicencesService.go' arguments instead of a variable, I was thinking could you do this for the very first test as well and then you only need to declare the filter variables in scope of its describe block on line 91

Reply:
I've made some changes to this to make it more consistent with the other tests